src/Tmds.MDns/ServiceBrowser.cs

```diff
@@ -149,7 +149,7 @@ internal void OnServiceRemoved(ServiceInfo service)
         }
 
         void OnNetworkInterfaceAdded(NetworkInterface networkInterface)
-        {            
+        {            

```

Comment:
whitespace change?

Reply:
This chage did not show up in my local diff viewer. Will try to omit it.